README.md

```diff
@@ -105,6 +106,32 @@ You can use any other LLM by invoking the same interface:
 llm = Langchain::LLM::GooglePalm.new(api_key: ENV["GOOGLE_PALM_API_KEY"], default_options: { ... })
 ```
 
+#### Groq OpenAI Compatible
+
+Add `gem "ruby-openai", "~> 7.0"` to your Gemfile.
+
+```ruby
+llm = Langchain::LLM::GroqOpenAI.new(api_key: ENV["GROQ_API_KEY"]) # Defaults to Meta Llama 3
```

Comment:
I would prefer to call it: `Langchain::LLM::Groq`. I find `GroqOpenAI` rather confusing as it sounds like the OpenAI models are available on Groq, which they're not.